circuit/src/aggregator/mod.rs

```diff
@@ -101,6 +101,18 @@ struct Aggregator {
 	as_proof: Option<Vec<u8>>,
 }
 
+impl Aggregator {
+	/// Create a new aggregator.
+	pub fn new(svk: Svk, snarks: Vec<Snark>, instances: Vec<Fr>, as_proof: Vec<u8>) -> Self {
+		Self {
+			svk,
+			snarks: snarks.into_iter().map_into().collect(),
+			instances,
```

Comment:
We dont need instances here. They are passed to the verifier as public inputs

Reply:
Ah, yes.
Will update it now!